src/js/components/organisms-page/content-cover/content-cover.html

```diff
@@ -0,0 +1,24 @@
+<article class="content-cover content-cover--cover">
+    <c-image v-if="this.image" v-bind="this.image"></c-image>
+
+    <div class="content-cover__image--overlay"></div>
+
+    <section class="content-cover__container">
+        <div class="flexgrid__cell--xs-6 content-cover-content content-cover--cover-reset">
+             <h3 class="content-cover__title">{{ title }}</h3>
+        <p class="content-cover__description" v-if="description">{{ description }}</p>
```

Comment:
`multiple components & v-if="cta ... & cta`
* why is cta mentioned twice in the v-if statement?
* same question for the other components below
* could this be a dynamic component using something like `<component v-if="cta" :is="cta.type" :class="getClasses(cta.type)"/>`?

Reply:
It should be solved better, I agree. But for the sake of speed i'm now adding it like this. We should create a proper component for this.

Reply:
added the change to branch